<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerChatWindow</class>
 <widget class="QMainWindow" name="ServerChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server Chat</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #f4f5f7;
}
QWidget#centralwidget {
    background-color: #f4f5f7;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="serverInfoSection" native="true">
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#serverInfoSection {
    background-color: #FFFFFF;
    border-right: 1px solid #E0E0E0;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>15</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>40</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="serverLogoLabel">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    background: #f2f4f8;
    border: 2px solid #e5e7eb;
    border-radius: 60px;
}</string>
       </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="serverOrgLabel">
        <property name="styleSheet">
         <string notr="true">font-size: 13pt; font-weight: 600; color: #1f2937;</string>
        </property>
        <property name="text">
         <string>Server Admin</string>
        </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="styleSheet">
          <string notr="true">QFrame {
    background-color: #E5E7EB;
    min-height: 1px;
    max-height: 1px;
}</string>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="serverIpLabel">
        <property name="styleSheet">
         <string notr="true">font-size: 10pt; color: #6b7280; padding: 6px; line-height: 1.4em;</string>
        </property>
         <property name="text">
          <string>Your IP Address:
Connection Status: Connected</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="styleSheet">
          <string notr="true">QFrame {
    background-color: #d0d0d0;
    min-height: 1px;
    max-height: 1px;
}</string>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="userListTitle">
        <property name="styleSheet">
         <string notr="true">font-size: 11pt; font-weight: 600; padding: 12px 8px 4px 8px; color: #6B7280;</string>
        </property>
         <property name="text">
          <string>Recent Conversations</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeft</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="userListWdgt">
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QListWidget {
    background-color: #ffffff;
    border: none;
    font-size: 11pt;
    padding: 6px;
    color: #111827;
}
QListWidget::item {
    background-color: transparent;
    border: none;
    border-radius: 10px;
    padding: 12px;
    margin: 4px 6px;
    color: #111827;
}
QListWidget::item:hover {
    background-color: #f0f4ff;
}
QListWidget::item:selected {
    background-color: #3b82f6;
    color: #ffffff;
}</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="restartServerBtn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Reset and Restart</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="chatSection" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#chatSection {
    background-color: #f4f5f7;
    border: none;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="chatHeader" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>70</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget#chatHeader {
    background-color: #FFFFFF;
    border-bottom: 1px solid #E5E7EB;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="chatTitleLabel">
            <property name="styleSheet">
             <string notr="true">font-size: 14pt; font-weight: bold; padding: 0px; color: #2C2C2C;</string>
            </property>
            <property name="text">
             <string>Public Chat</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="chatSubtitleLabel">
            <property name="styleSheet">
             <string notr="true">font-size: 10pt; padding: 2px; color: #666666;</string>
            </property>
            <property name="text">
             <string>Your messages will be sent to everyone</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="chatHistoryWdgt">
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QListWidget {
    background-color: #f4f5f7;
    border: none;
    font-size: 13pt;
    padding: 12px;
    color: #111827;
}
QListWidget::item {
    background-color: transparent;
    border: none;
    padding: 0px;
    margin: 0px;
}
QListWidget::item:selected,
QListWidget::item:selected:active,
QListWidget::item:selected:!active,
QListWidget::item:selected:hover {
    background-color: transparent;
    border: none;
    outline: none;
}</string>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
         </property>
         <property name="spacing">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="inputSection" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget#inputSection {
    background-color: #ffffff;
    border-top: 1px solid #e5e7eb;
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QTextEdit" name="typeMessageTxt">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>60</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LayoutDirection::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QTextEdit {
    background-color: #ffffff;
    color: #111827;
    border: 1px solid #e5e7eb;
    border-radius: 24px;
    padding: 14px 18px;
    font-size: 13pt;
}
QTextEdit:focus {
    border: 2px solid #3b82f6;
}</string>
            </property>
            <property name="placeholderText">
             <string>Type your message...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sendFileBtn">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>ï¼‹</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="recordVoiceBtn">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>ðŸŽ¤</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sendMessageBtn">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>60</height>
             </size>
            </property>
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
